app/templates/email_templates/email_user_request_removed.html

```diff
@@ -3,7 +3,7 @@
 
     <p>{{ name }} has been removed from the request, <a href='{{ page }}'>{{ request_id }}</a>.</p>
 {% else %}
-    <p>Attention {{ agency_name }} Users,</p>
+    <p>Attention {{ name }},</p>
 
     <p>{{ name }} (You) has been removed from the request, <a href='{{ page }}'>{{ request_id }}</a>.</p>
```

Comment:
Maybe it would be better like this:
`You ( {{name}} ) have been removed...`

"(You) has" might lead to complaints by idiots.
If this has already been approved by Pauline, then I guess we can ignore it...